.github/workflows/build-image.yml

```diff
@@ -0,0 +1,36 @@
+name: Build Image
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+
+  workflow_dispatch:
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+
+    container:
+      image: quay.io/centos-bootc/builder:latest
```

Comment:
General comment:  I noticed that quay.io/centos-bootc/builder doesn't provide any information yet.  Long-term it could be useful to drop some README there and point to the source repositories on GitHub.